telebot/main.py

```diff
@@ -0,0 +1,37 @@
+import constants as keys # can move the api_key to .env file alternatively
```

Comment:
Yup, as you have pointed out, please refrain from posting the API key on this repo becos it is public. You can send it to me on a more secure platform. I have gone ahead to delete that file